File: Exercicios_guanaraba/Com_Ruby/exercicios/ex018_Seno_Cosseno_Tangente.rb
=begin

Faça um programa que leia um ângulo qualquer e mostre na tela o valor 
do seno, cosseno e tangente desse ângulo.

=end
angulo = gets.chomp.to_f

angulo = (angulo/180 * Math::PI)

puts "Seno :#{Math.sin(angulo).truncate(2)}"

puts "Cosseno :#{Math.cos(angulo).truncate(2)}"

puts "Tangente :#{Math.tan(angulo).truncate(2)}"

File: Exercicios_guanaraba/Com_Dart/bin/ex018_Seno_Cosseno_Tangente.dart
/*

Faça um programa que leia um ângulo qualquer e mostre na tela o valor do seno, cosseno 
e tangente desse ângulo. 

*/
import 'dart:io';
import 'dart:math';
void main(){

  double angulo = double.parse(stdin.readLineSync());

  angulo = (angulo/180 * pi);

  print ('Seno :' + sin(angulo).toString());

  print ('Cosseno :' + cos(angulo).toString());

  print ('Tangente :' + tan(angulo).toString());
  

}